docs/070-project-management/pm-unanet-tasks.md

```diff
@@ -0,0 +1,246 @@
+   - Select the plans you wish to save by clicking on the "Assign" box at the beginning of each plan (or the one above to "assign all")
+
+### Tips
+
+You may enter multiple hours/month on the planning grid rather than one month per person at a time. You can do this multiple ways.
+
+1. For the same number of hours for every person for every period - Double click on the space to the left of the first month in the header
+
+![Screenshot of Plan with Grid 6](../images/Plan-with-Grid-6.png)
+
+- You will be directed to the following screen where you can "enable budget field for updates" and enter the hours you wish to allocate per person per month.
+- You can also enter the labor category here if it is the same for each person.
+
+![Screenshot of Plan with Grid 7](../images/Plan-with-Grid-7.png)
+
+2. For the same number of hours for an individual for every period - Double click on the name of the person
+
+   - You will be directed to a similar screen where you can "enable budget field for updates" and enter the hours you wish to allocate for that person for each month.
+   - You can also enter the labor category here.
+
+3. For the same number of hours for each person for a particular month - Double click on the month at the top of the page
+
+   - You will be directed to a similar screen where you can "enable budget field for updates" and enter the hours you wish to allocate for each person for the selected month.
+   - You can also enter the labor category here if it is the same for each person
+
+## Adjust projections for holidays or other time-off
+
+In general, resource allocations can be expected using the following rations:
+
+1. If a FT resource devoted to the project, then 80%
+2. If an AD resource devoted to the project, then 65%
+3. If a PT resource devoted to the project, then 40%
+
+From that place, then deduct - monthly - for any of the following:
+
+1. holidays
+2. OOO
+3. Any other events, e.g., summits, etc.
+
+Pro Tip: For finding OOO dates on the calendar, use the CivicActions shared calendar and try searching "name OOO". This will allow you to isolate the forthcoming dates for the given person.
+
+## Reviewing and approving expenses
+
+### Find your approvals
+
+1.  Click on the Unanet logo
+1.  Scroll down to Approvals
+1.  Look for Expense Reports and/or Requests and click on Primary Approvals to view
+1.  ![Screenshot of Approving Expense 1](../images/Approving-Expenses-1.png)
+1.  NOTE: All expense reimbursements except for ProDev < $50 should be submitted as a REQUEST first.
+1.  To check this quickly:
+    1. Click on the magnifying glass next to the expense report
+    1. Scroll down to Approval History
+    1. A report that has previously been a request will show "PreApproving" and "PreApproved" statuses in the history. If it has never been a request, you will only see "Approving" and/or "Completed". (See below for explanation of statuses and an example of an expense request and report approval history).
```

Comment:
@farooqzakhilwal one last change, reduce the indentation here my one space and that should fix it. If you look at https://handbook.civicactions.com/en/farooqzakhilwal-patch-1/070-project-management/pm-unanet-tasks/ you can see that the sub-list is all one line instead of a list.